### ISSUE TITLE
the defensive height bonuses are to much

### ISSUE THREAD

**Issue by Swartonf** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#918]
I'm not sure if the suggestion has ever been made, but I wanted to ask you for your opinion on the two defensive height bonuses.

1.) If you are on an increase (like tower or wall), there is a defensive bonus x3 for the units
2.) Depending on which Tower / Wall / Keep ... it is, there is also a miss function for attacking with ranged fighters of up to 80%

If a unit is standing on a round tower, you have to shoot an arrow 15 times as often as if it were standing on the level. Therefore 1 archer on the tower defeats 4 archers on the land.
I find this amount to be an exaggeration. The result is that AI fights are very limited, with a 2 great defence bonuses it will always be a tie if you dont restrict the fights like the creaters had to do in the AI-Tournament.qq
In the Human vs AI game, it simplifies the game too much as well, since it's all about defending the first attacks until you have a good offensive and the defence itself is to easy.

My suggestion would be that the defensive high-bonus of x3 is removed or reduced. Then there is still the (f.e.) 80% miss bonus on the big tower. But with this kind of bonuses, the player defence is to easy and the AI-Defence is to strong for AI's to counter

**Comment by Monsterfisch** [comment]
The real problem in the ai matches is the units that are stacked on the keep tbh it has little to do with the defensive bonuses of defense structures. If there was a way to spread the ranged troops over the castle, instead of on the keep together with the melee units, this would prevent a death ball from forming on there (melee units tank a lot of hits for the ranged troops making it so that the lord is almost unapproachable). This way the ranged units would be spread thinner over the castle plus the risk for the units to die when the tower is destroyed by siege etc. granted if you have 200 ranged troops on a single tower it's not particularly easier to approach but it's definitely more vulnerable. then on an indestructible keep. As for the player vs ai, it's more a matter of using siege weapons for the ai. If it was possible for ai to use them more proficiently, it would already do a lot of work to make later sieges more interesting.

**Comment by OverlordMatty** [comment]
Yep, defender has too much of an advantage. +1

**Comment by GRhin** [comment]
Considering portable shields completely nullify this bonus, no I don't think the height advantage is too much. 
Problem is ai handle shields too poorly which I would say is the better solution to the problem

**Comment by GRhin** [comment]
Not to mention trebs outrange and can destroy those towers and walls is a pretty good counter balance

**Comment by Swartonf** [comment]
I agree with the points u made. Catapults nullify towers, except for the keep. The real problem is, that the AI doesnt know how to use catapults or shields properly.

This however, seems to be out of reach at the moment and its a complex feature to add

Thats why we can also work around that problem. At the moment, the ranged attackers die to fast and the melees have a very hard time to go through the castle without any ranged backups. If the defenders dont have the 3 times defence bonus anymore, they will die as well. The ranged attackers survive longer and the meeles will have their backup they need. 

Even if that wasnt the case, a ### **15times**  defence boost (with the miss-function), just because some1 is standing on a tower, is way to much.

**Comment by GRhin** [comment]
I have never defended from an ai and thought "I think I'm only surviving
because of height advantage".
But even if that were true, if your goal is to make ai better (which is a
huge task), doing it in a way that also punishes them (when they are
defending) is not the way to go about it in my opinion.
Basically either your issue is height advantage, in which case my previous
message details why I disagree, or your issue is that ai are too weak,
which I don't think this change solves effectively and would prefer to make
AIs troops stronger in general (make them 3x as strong it does what you
want for when they attack, and improves defense, I wouldn't go that far,
maybe 2x max)

15 times defense boost is not toouch considering how simple it is to make
that play out as 0 boost, in fact potentially a negative depending on your
seige ability.

**Comment by gynt** [comment]
So, only have the defensive bonus for AI's ? :) That seems like a fun "hard-mode" experience people would try.

**Comment by GRhin** [comment]
No, I don't think you understood.
I would do just ranged units for this.
Basically you increase the base damage human owned units take from ranged
units, that way the damage done when they have high ground is closer to the
original base damage, but in all other cases they take more damage still.
For example, if you make them take double damage, so archers, which
normally take 10,000 damage from arrows, now take 20,000. If they are on
walls, they take 6,666 damage, but on ground they still take that full
20,000
